dmscripts/mark_definite_framework_results.py

```diff
@@ -0,0 +1,135 @@
+from collections import Counter
+import logging
+
+
+import jsonschema
+
+
+def _passes_validation(candidate, schema, logger, schema_name="schema", tablevel=0, loglevel=logging.INFO):
```

Comment:
After initially  complaining about passing a logger around you've really gone to town here with the tablevel and everything :)